save-plugins/fix-plugin/README.md

```diff
@@ -46,9 +46,11 @@ exec_cmd="./ktlint -R diktat-0.4.2.jar -F"
 testFilePattern="*Test.kt"
 expectedFilePattern="*Expected.kt"
 batchMode = false
+resourceNamePattern = """(.+)(Expected|Test)\.[\w\d]+"""
```

Comment:
Please check correct toml syntax for this; what's the difference between usual and triple-quoted string and whether slashes need escaping